src/main/java/com/iota/iri/model/Hash.java

```diff
@@ -3,6 +3,9 @@
 import com.iota.iri.crypto.Curl;
 import com.iota.iri.storage.Indexable;
 
+/**
+ * Represents an indexable hash object
+ */
 public interface Hash extends Indexable, HashId {
     
     Hash NULL_HASH = HashFactory.TRANSACTION.create(new byte[Curl.HASH_LENGTH]);
```

Comment:
This is a public element so add documentation here please (it is the hash of the genesis transaction)